src/NewsService.Data.Provider.MsSql.Ef/Migrations/20210906170536_AddPreviewColumn.cs

```diff
@@ -0,0 +1,19 @@
+﻿using LT.DigitalOffice.NewsService.Models.Db;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace LT.DigitalOffice.NewsService.Data.Provider.MsSql.Ef.Migrations
+{
+  [DbContext(typeof(NewsServiceDbContext))]
+  [Migration("20210906170536_AddPreviewColumn")]
+  public class AddPreviewColumn : Migration
+  {
+    protected override void Up(MigrationBuilder migrationBuilder)
```

Comment:
also write down method

Reply:
done

Reply:
strange, in the studio changed, but in git it remained